Thunder Morpho

dbosen opened the pull request:
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.
- [ ] New features or changes are documented.

If you are really awesome, then your feature is covered by additional tests. Well done!

dbosen committed:
reset changelog

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  fix: form alter theme check

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Issue #3229926 by fago, volkerk: Allow install admin_toolbar v3, require security update

chrfritsch committed:
feat: enable gin theme 

* feat: enable gin theme

* fix: add form dsescription toggle

* test: fix some tests

* fix: add blocks

* fix: add sorting option

* test: fix

* fix: add sidebar patch

* test: open sidebar

* test: fix open sidebar

* test: disable autosave form notifications in tests

* fix: remove patch

* fix: add favicon

chrfritsch committed:
Ad thunder version to status page

Co-authored-by: Volker Killesreiter <killesre@gmail.com>

chrfritsch committed:
feat: gallery preview in paragraphs

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Fix ajax test errors
  build(deps): bump nth-check from 2.0.0 to 2.0.1 (#430)
  build(deps): bump prismjs from 1.24.1 to 1.25.0 (#429)
  npm package update

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  fix: config changes

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Use paratest in tests (#436)
  Test with composer2 in upgrade test

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Fix selenium service and use latest selenium version.
  Selenium in docker
  Language in graphql requests
  Issue #3243283 by daniel.bosen, chr.fritsch: Release Thunder 6.2.2
  Change existing config only
  test: check for deprecations (#439)

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Update field group module
  Update npm

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Use "Create a fault-tolerant method for interacting with links and fields in Javascript test." patch
  Add option to skip test cleanup
  Update npm packages

chrfritsch committed:
Merge branch '6.2.x' into morpho

* 6.2.x:
  Issue #3250864 by volkerk, chr.fritsch: Add base theme determination
  Update php versions in tests
  Issue #3250590 by daniel.bosen, chr.fritsch: Release Thunder 6.2.3
  Issue #3250583 by daniel.bosen, chr.fritsch: GraphQL responses contain https links

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  feat: parallel testsuites
  chore: update json-schema
  Drupal 9.3 compatibility

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  fix: disable fast failing for GH actions

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Issue #3250638 by eakroko: GraphQL undefined index error

chrfritsch committed:
fix: morpho tests

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Refactor and reorganize javascript test traits
  Phpstan update

chrfritsch committed:
Merge branch 'morpho' of github.com:thunder/thunder-distribution into morpho

* 'morpho' of github.com:thunder/thunder-distribution:
  fix: morpho tests

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  phpstan: do not extend from internal classes
  Issue #3254295 by Volker23, daniel.bosen: Facets module < 2.0 not supported
  docs: reflect new function signature
  Issue #3253753 by szeidler, shagel: Plain text twig filter throws an error if field does not exist or is hidden (#481)
  test: wait for selector
  Coding style checks also for install files
  GraphQL entity links producer access check
  Issue #3253462 by chr.fritsch, daniel.bosen: Release Thunder 6.3.0
  Sort all config files properly

ol0lll committed:
Merge branch '6.3.x' of github.com:thunder/thunder-distribution into morpho

ol0lll committed:
remove collapsible sidebar gin patch to make installable

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Issue #3259071 by chr.fritsch, daniel.bosen, volkerk: Enable new paragraphs_features quick links functionality
  Issue #3255519 by useernamee, jjchinquist, golubovicm, vasike, chr.fritsch: Conflict with hierarchy manager
  fix: thunder tests
  Issue #3244796: The module thunder_riddle is missing after upgrading from thunder 3 to 6
  Add language type parameter for ThunderLanguage DataProducer
  fix: paragraphs feautres tests
  Remove mention of instant articles.
  Phpstan level 4
  Update vuepress
  Keep Drupal version within minor releases on updates.
  Switch from own our subrequest producer to graphql
  Fix composer installed binaries.

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Issue #3260558 by daniel.bosen, chr.fritsch: Release Thunder 6.3.3
  refactor: phpstan lvl 5
  Allow update helper v3
  Issue #3239013 by BAHbKA, chr.fritsch, daniel.bosen: UX: Thunder media library ignores field cardinality
  Fix ThunderTestSuite
  Issue #3260068 by daniel.bosen, chr.fritsch: Release Thunder 6.3.2
  Issue #3260090 by daniel.bosen, chr.fritsch: Invalid token used in metatag config
  Fix slick config
  Issue #3259163 by chr.fritsch, daniel.bosen: Add structured data to Thunder article by a JSON-LD schema

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Phpstan lvl 6

chrfritsch committed:
Fix morpho test

* test: fix tests

* fix: sroll to element

* fix: entity browser is broken. skip the test

* fix: stan

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  refactor: move test methods
  Fix new drupal coder rule

chrfritsch committed:
Merge branch 'morpho' of github.com:thunder/thunder-distribution into morpho

* 'morpho' of github.com:thunder/thunder-distribution:
  Fix morpho test

chrfritsch committed:
fix: tests after merge

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  fix: remove blazy conf
  Graphql producer annotation
  Allow all status codes

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  fix: remove graphql patch

chrfritsch committed:
Issue #3067847: [META] Switch from Entity Browser to Media Library

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Unpin paragraphs
  build(deps): bump prismjs from 1.26.0 to 1.27.0
  Require php 7.4
  Add mappable conditions to entites with term data producer
  Update VBO module to 4.x
  Issue #3265222 by chr.fritsch, daniel.bosen: PHP8.1 compatibility
  Issue #3264904 by daniel.bosen, chr.fritsch: Release Thunder 6.3.4

chrfritsch committed:
fix: gin.seetings

chrfritsch committed:
fix: add patch for buttons in modal

chrfritsch committed:
Revert "fix: add patch for buttons in modal"

This reverts commit 1cde10d54b24a82f7319d04ec268f7fb12277b06.

chrfritsch commented:
We need https://www.drupal.org/project/drupal/issues/3151534, but it currently breaks the tests

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Fix README.md contributing guidelines link

chrfritsch committed:
Update path for morpho

* feat: update-path

* fix: upgrade

* fix: add update_helper patch

* empty

* fix: diff test

* fix: tests

* fix: configs

* empty

* fix: finalize

* fix: cs

* fix: remove one update

chrfritsch committed:
Switch to legacy toolbar

* feat: use gin_toolbar

* empty

* Revert "empty"

This reverts commit 2e0efa4d07713fe0dfcf4e6280eb12c9a1e3d27f.

* fix: tests

* fix: use legacy toolbar

* fix: scroll to element

chrfritsch committed:
Remove liveblog integration

chrfritsch committed:
Remove thunder admin classes 

* fix: remove old claro classes

* empty

chrfritsch committed:
Show sidebar on term and media add/edit pages

* fix: show sidebar on term pages

* fix: add return type

* fix: aÃlso use for media items

* fix: check for standalone url

* fix: switch hook

* fix: add comment

chrfritsch committed:
Merge 6.3.x

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Issue #3269598 by daniel.bosen, chr.fritsch: Release Thunder 6.3.5
  Issue #3269389 by daniel.bosen, chr.fritsch, shagel: Improved basic page

chrfritsch committed:
fix: remove dropzone patch

chrfritsch committed:
feat: remove better normalizers

* feat: remove better normalizers

* empty

chrfritsch committed:
fix: move to post updates (#534)

* fix: move to post updates

* fix: update firs

* fix: return type

* fix: require

* empty

* empty

* empty

* empty

* empty

* empty

* empty

* empty

* empty

* empty

* empty

* fix: permissions removal

* empty

* fix: another update

* empty

* empty

* empty

* empty

chrfritsch committed:
feat: remove adsense module

* feat: remove adsense module

* Issue #3270689 by eakroko, chr.fritsch: GraphQL Menus are not working for views

* Extract thunder_gqls test base class functionality into a trait

* Extract test trait

* Anonymous user already has all permissions, when thunder_gqls module is enabled

* remove login in jsonld test

* get http client

* use BrowserHtmlDebugTrait

* Update testing.md

* Revert "feat: remove adsense module"

This reverts commit 4a16a769eab5fa318b462e6bc7d5c2f7671510f2.

* Revert "Extract thunder_gqls test base class functionality into a trait"

This reverts commit 04ccb6997107819f22c1c2e6430beea93f21761b.

* Revert "Issue #3270689 by eakroko, chr.fritsch: GraphQL Menus are not working for views"

This reverts commit cb4351e471292044eda25cb38cecf51ccad3af59.

* Revert "Revert "feat: remove adsense module""

This reverts commit 85eca5601c5700250635ebc3c4ca92a5726ef100.

Co-authored-by: ekrokowski <e.krokowski@trio-group.de>
Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  Allow checklist v2
  Extract thunder_gqls test base class functionality into a trait
  Issue #3270689 by eakroko, chr.fritsch: GraphQL Menus are not working for views

dbosen committed:
Merge branch '6.3.x' into morpho

dbosen committed:
Remove Google Analytics Module

dbosen committed:
Merge remote-tracking branch 'origin' into morpho

dbosen committed:
Merge remote-tracking branch 'origin' into morpho

dbosen committed:
Merge remote-tracking branch 'origin' into morpho

dbosen committed:
Fix paragraphs paste testing.

dbosen committed:
Remove shariff

dbosen committed:
Merge remote-tracking branch 'origin' into morpho

chrfritsch committed:
fix: remove table sorting

* fix: remove table sorting

* empty

chrfritsch committed:
fix: rename node form class

chrfritsch committed:
Merge olivero

* Update blazy and slick min versions

* Issue #3278186 by daniel.bosen, volkerk: Release Thunder 6.3.7

* Issue #3281046 by chr.fritsch, daniel.bosen: Use olivero as Thunder's new frontend theme

Co-authored-by: Volker Killesreiter <killesre@gmail.com>
Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>

chrfritsch committed:
Merge branch '6.3.x' into morpho

* 6.3.x:
  fix: default config export
  Issue #3281046 by chr.fritsch, daniel.bosen: Use olivero as Thunder's new frontend theme
  Issue #3278186 by daniel.bosen, volkerk: Release Thunder 6.3.7
  Update blazy and slick min versions

ol0lll committed:
Initial article form styling

* Style group teaser

* basis group

* add section class for "Basis", additional layouting

* add config update

chrfritsch committed:
Fix paragraph preview

dbosen committed:
Forward ports of 6.3.x changes

dbosen committed:
Remove thunder base and thunder admin themes

dbosen committed:
Keeping up with 6.3.x (#578)

* Fix responsive image style dependency

* fix: phpstan config

* fix: phpstan config

* cleanup config

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Issue #3281562 by eakroko, daniel.bosen: GraphQL Menus are not working for 404 urls

* Home was replaced with back to site.

* we do not want to test oliveros breadcrumb.

* Breadcrumb is now pointing to node view on edit pages

* code style fixes

Co-authored-by: Christian Fritsch <chr.fritsch@gmx.net>
Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: ekrokowski <e.krokowski@trio-group.de>

chrfritsch committed:
Drupal 9.4 compat

* Fix responsive image style dependency

* fix: phpstan config

* fix: phpstan config

* cleanup config

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Issue #3281562 by eakroko, daniel.bosen: GraphQL Menus are not working for 404 urls

* Issue #3291454 by daniel.bosen, chr.fritsch: Drupal 9.4 compatibility

* fix: add config changes

* fix: deprecated code

* Set phpstan baseline

* Issue #3291445 by daniel.bosen, chr.fritsch: Release Thunder 6.4.0

* Changelog and version bump

* Remove patch

* generate new baseline

* fix config

* back to 80

* no update

Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>
Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: ekrokowski <e.krokowski@trio-group.de>

chrfritsch committed:
Merge 6.4.x changes back into morpho

ol0lll committed:
Merge branch '6.4.x' of github.com:thunder/thunder-distribution into morpho

ol0lll committed:
Fix phpstan baseline

chrfritsch committed:
Merge pull request #601 from thunder/fix-morpho

Fix morpho

chrfritsch committed:
Update morpho 

* Fix export of GraphQL extensions

* Fix phpstan errors (#606)

* Fix phpstan errors

* Actually return an int

Co-authored-by: Alex Pott <alex.a.pott@googlemail.com>

* fix/instagram-graphql-type (#605)

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Remove password policy patch, update diff module and increase selenium version

* remove password policy patch

* article

* revert

* selenium version

* new diff version

* update entity browser

* selenium 105

* remove diff patch

* do not update entity browser

* Remove test-performance action

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Remove gin patch

Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>
Co-authored-by: Volker Killesreiter <killesre@gmail.com>
Co-authored-by: Alex Pott <alex.a.pott@googlemail.com>
Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Update simple_sitemap module (#612)

* Update simple_sitemap module

* Update config

* Update test

* Update config

* Fix test

* Add some docs

* Fix config

* empty

* Revert "empty"

This reverts commit c6a27f9ad6f07e448c48057e83714ee2f930b3e3.

* Update core

* Update modules

* Update simple_sitemap

* Update simple_sitemap.sitemap.index.yml

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

Commit:
Merge branch '6.4.x' into update-morpho-11-2022

* 6.4.x:
  Update field_group module (#613)
  Issue #3319701 by alexpott, daniel.bosen: Add Media File Delete integration and new permission to delete any file
  Remove test-performance action
  Remove password policy patch, update diff module and increase selenium version
  fix/instagram-graphql-type (#605)
  Fix phpstan errors (#606)
  Fix export of GraphQL extensions

Commit:
Fix

Commit:
Fix test

chrfritsch committed:
Merge pull request #615 from thunder/update-morpho-11-2022

Update morpho 11 2022

ol0lll committed:
Enable gin modern sidebar (#603)

* enable modern sidebar

* enable secondary_toolbar_frontend

* Device switcher is not accessible from overly when using gin secondary toolbar.

* Remove patch

* f

* remove config changes

chrfritsch committed:
Pull 6 4 changes 

* Update password_policy module

* Remove php 7.4 testing (#618)

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Fix slick tests (#625)

* Fix slick tests

* Fix cs

* Fix

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Vgwort (#624)

* composable schema for vgwort

* wip

* code styles

* phpcs

* vgwort can be null

* A test

* cs

* white space fix

* fix test

* Add some documentation

* Markdown style fixes

* Issue #3328054 by daniel.bosen, chr.fritsch: Release Thunder 6.4.4

* Bump qs from 6.5.2 to 6.5.3 (#622)

Bumps [qs](https://github.com/ljharb/qs) from 6.5.2 to 6.5.3.
- [Release notes](https://github.com/ljharb/qs/releases)
- [Changelog](https://github.com/ljharb/qs/blob/main/CHANGELOG.md)
- [Commits](https://github.com/ljharb/qs/compare/v6.5.2...v6.5.3)

---
updated-dependencies:
- dependency-name: qs
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>
Co-authored-by: Christian Fritsch <chr.fritsch@gmx.net>

* Cleanup labels (#628)

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Remove schema_metatag patch (#627)

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

* Update scheduler module

* Remove split text feature

* Remove split feature tests

* Fix config

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
Merge branch '6.4.x' into upgrade-morpho-jan-2

* 6.4.x:
  remove robo require
  Fix upgrade test after composer 2.5 release
  Update scheduler module
  Remove schema_metatag patch (#627)
  Cleanup labels (#628)
  Bump qs from 6.5.2 to 6.5.3 (#622)
  Issue #3328054 by daniel.bosen, chr.fritsch: Release Thunder 6.4.4
  Vgwort (#624)
  Fix slick tests (#625)
  Remove php 7.4 testing (#618)
  Update password_policy module

chrfritsch committed:
Merge pull request #635 from thunder/upgrade-morpho-jan-2

Upgrade morpho jan 2

chrfritsch committed:
Drupal 10

ol0lll committed:
Fix field action displayed as modal action 

* Fix field action displayed as modal action

* Update composer.json

Co-authored-by: Christian Fritsch <chr.fritsch@gmx.net>

ol0lll committed:
Enable select2 on media library original media edit form

chrfritsch committed:
Issue #3331225 by chr.fritsch, volkerk: Release Thunder 6.5.0 (#641)

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Update config and patches 

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Do not use contextual overrides for media items 

* Do not use contextual overrides for media items

* Remove patch

* Remove docs

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Increase upgrade source

* Increase upgrade source

* fix yml

* Update package

* Fix lock

* Update diff

* Uodate core-dev

* Require seven and ckeditor

* New update file

* New dump structure

* Add dependencies

* Lets see

* Try

* Install modules

* remove logs

* Remove shariff field

* Revert translatable

* Update lock

* try

* Revert "try"

This reverts commit 66f87952d331ed1f6a7c0486d770a7380a29531a.

* Revert "Update lock"

This reverts commit 99f5cddabfe0e1acb608f95b89d7e104945822c2.

* New try

* try

* Use checkout v3

* Fix typo

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>

chrfritsch committed:
Remove edit link from media library view 

* Remove own media library view

* Add view

* Remove dependency

* media library view same as thunder 6

* filters same as thunder 6

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: Volker Killesreiter <killesre@gmail.com>

ol0lll committed:
Remove fieldgroup labels on entity forms

chrfritsch committed:
Add Gin patches (#652)

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Update responsive preview

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Streamline image styles

* Streamline image styles

* Move to install

* Add thumbnail style

* Show more images in gallery preview

* Add update hook

* empty

* Readd twitter and facebook img style

* Revert image styles in focal point widgets

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Always use gin light mode

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

chrfritsch committed:
Remove ctools dependency

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

ol0lll committed:
borders and spacing

* borders and spacing

* remove margin reset for sections

---------

Co-authored-by: Christian Fritsch <chr.fritsch@gmx.net>

dbosen committed:
Merge remote-tracking branch 'origin/6.5.x' into merge-latest-6.5.x

dbosen committed:
add media entity pinterest patch

chrfritsch committed:
Merge pull request #662 from thunder/merge-latest-6.5.x

Merge latest 6.5.x

chrfritsch committed:
Disable paratest because it skips all tests

ol0lll committed:
Fix morpho heroku

chrfritsch committed:
No edit on multiple image upload

* No edit on multiple image upload

* remove patches

* up min versions

* Skip the edit form

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: Volker Killesreiter <killesre@gmail.com>

chrfritsch committed:
Show media library buttons side-by-side

* Show media library buttons side-by-side

* Add css

* fix buttons

* use version

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: Volker Killesreiter <killesre@gmail.com>

dbosen committed:
Content moderation edit tweaks

* move buttons

* refactoring

* Buttons all over the place

* move stuff to thunder_workflow

* remove custom css

* cleanup

* cs fixes

* fix getentity

* final constructor

* final constructors

* Fix core patch

* remove filter

* Fix tests

* Use new image style

* refactor

* Cbf

* return types

* Add return type

* empty

* remove transparent

* less specific

* Fix upgrade path

* Better conditions

* moderated only

* we can be sure to have content moderation available

* try tests for move to actions

* fix stan

* add tests

* Correctly declare required parameters

---------

Co-authored-by: Christian Fritsch <chr.fritsch@gmx.net>
Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

Commit:
Merge branch '6.5.x' into merge-65-into-morpho

* 6.5.x:
  Fix paragraphs IEF warning
  Release 6.5.3
  Use field data producer
  Issue #3343922 by daniel.bosen, chr.fritsch: Release Thunder 6.5.2
  Issue #3340586 by dejan0: Attempted to create an instance of field with name field_meta_tags on entity type node when the field storage does not exist

ol0lll committed:
fix test

ol0lll committed:
Fix paragraphs warning icon position and size

ol0lll committed:
Merge branch 'morpho' into merge-65-into-morpho

chrfritsch committed:
Merge latest 6.5 into morpho

Merge latest 6.5 into morpho

chrfritsch committed:
Fix focal point preview closes media edit form

* Fix focal point preview closes media edit form

* Remove patch

* remove new line

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

dbosen committed:
Remove linkit patch

* remove patch

* beta4 min version

* Fix core version

ol0lll committed:
Split article css into separate libs

* initial

* align with changes in 6.5.x

* fix

* revert file

* fix cs

* fix stan

* remove groups on gallery para preview

* trigger

dbosen committed:
Fix return on form

* wip

* simplify

* use context

* Fix stuff

* lint

* add date

* add once to globals

* test should fail

* test new test only

* click'n'run

* Revert "click'n'run"

This reverts commit cb9525b91fc2ae9dc206fc6f573bd3fae79a0f53.

* streamline

---------

Co-authored-by: Volker Killesreiter <killesre@gmail.com>

dbosen committed:
Webp support 

* Allow webp

* tests

* ThunderJavascriptTrait

* try direct call

* docs

* cbf

* try different test

* cs

* run all tests

* move test

* cs fixes

* add webp

* use jpg

* fix id

* nw try with webp

* assert media library was closed

* check for text

* fix

* move to own file

* test new file

* remove pragraphs test trait

* baseline...

* fail test

* unfail

* run all tests

* improved documentation

* rename function

---------

Co-authored-by: Christian Fritsch <chr.fritsch@gmx.net>

chrfritsch committed:
Add url field to social paragraphs

* Add url field to social paragraphs

* Add update hook

* Add field groups

* Add update hook for groups

* empty

---------

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>
Co-authored-by: Daniel Bosen <daniel.bosen@burda.com>

chrfritsch committed:
Update actions cache and upload

Co-authored-by: Christian Fritsch <christian.fritsch@burda.com>

ol0lll committed:
Multiple layout fixes

* Show upload errors without hovering.
* Show thunder logo in admin toolbar.
* Fix media library widget layout in teaser section.
* Bulk ops missing background in firefox esr.

dbosen committed:
Update docs
